src/cli/cmd/query/modules/modules_test.go

```diff
@@ -56,7 +56,7 @@ func Test_CmdModules(t *testing.T) {
 				cmd.SetArgs(args)
 			},
 			ValidateOutput: func(ctx context.Context, t *testing.T, cmdOpts clitesting.CmdOpts, output []byte) bool {
-				expectedOutput := "{\"modules\":[{\"id\":\"" + mockUuid1.String() + "\",\"taxonomyId\":\"00000000-0000-0000-0000-000000000000\",\"moduleName\":\"Rds\",\"source\":\"/Users/xyz/abc/tf-dir\",\"version\":\"1\",\"description\":\"\",\"inputAttributes\":[],\"namespace\":\"" + config.FarmDefault() + "\"}],\"page\":{\"size\":100,\"index\":0,\"total\":0}}"
+				expectedOutput := "{\"modules\":[{\"id\":\"" + mockUuid1.String() + "\", \"taxonomyId\":\"00000000-0000-0000-0000-000000000000\", \"moduleName\":\"Rds\", \"source\":\"/Users/xyz/abc/tf-dir\", \"version\":\"1\", \"description\":\"\", \"inputAttributes\":[], \"namespace\":\"" + config.FarmDefault() + "\"}], \"page\":{\"size\":100, \"index\":0, \"total\":0}}"
 				return assert.Equal(t, expectedOutput, string(output))
```

Comment:
If this is JSON it should probably use assert.JSONEq() instead.

Reply:
Yes, that's a good point. assert.JSONEq() handles differences in spacing.